lib/ui/src/core/addons.js

```diff
@@ -22,14 +22,18 @@ export default ({ provider, store }) => {
       return ensurePanel(api.getPanels(), selectedPanel, selectedPanel);
     },
     setSelectedPanel: panelName => {
-      store.setState({ selectedPanel: panelName });
+      store.setState({ selectedPanel: panelName }, { persistence: 'session' });
     },
   };
 
   return {
     api,
     state: {
-      selectedPanel: ensurePanel(api.getPanels()),
+      selectedPanel: ensurePanel(
+        api.getPanels(),
+        store.getState().selectedPanel,
+        store.getState().selectedPanel
```

Comment:
This is a bit weerd (grabbing the selected panel twice). Is it intentional?

Reply:
Yeah, what's going on here?

Reply:
Just followed the arguments of `ensurePanel`,
I can remove 1 I think.